descriptors/ModuleDescriptor-template.json

```diff
@@ -654,6 +654,37 @@
         }
       ]
     },
+    {
+      "id": "orders-storage.bind-piece-items",
```

Comment:
thinking need to change contract to bind-item-pieces
Because we can have 1 item and multiple pieces associtated to that item,
![image](https://github.com/folio-org/mod-orders-storage/assets/25097693/82b50bcb-e2bf-4483-acc4-78ab299f1c83)